javascript/code-challenges/challenges-05.test.js

```diff
@@ -0,0 +1,368 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function that iterates over an array of people objects
+and creates a new list of each person's full name using the array method 'map'.
+Each object will have the shape {firstName:string, lastName:string}
+E.g. [ { firstName:"Jane", lastName:"Doe" }, { firstName:"James", lastName:"Bond"}]
+should convert to ["Jane Doe", "James Bond"]
+Note the space in between first and last names.
+You can assume that neither firstName nor lastName will be blank
+------------------------------------------------------------------------------------------------ */
+const toLastNames = people => {
+  return people.map(person => person.firstName + ` ${person.lastName}`);
```

Comment:
This is kind of a funny use of concatenation and template literals. I'd say use one or the other, but my preference is template literals.
```js
person => `${person.fiirstName} ${person.lastName}`;
```